docs/team/yijiano.md

```diff
@@ -0,0 +1,120 @@
+      [#26](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/26),
+      [#27](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/27),
+      [#28](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/28),
+      [#29](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/29),
+      [#30](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/30),
+      [#31](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/31),
+      [#32](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/32),
+      [#33](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/33),
+      [#34](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/34),
+      [#35](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/35),
+      [#36](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/36),
+      [#37](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/37),
+      [#38](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/38),
+      [#40](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/40),
+      [#41](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/41),
+      [#47](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/47),
+      [#59](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/59),
+      [#60](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/60),
+      [#61](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/61),
+      [#62](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/62),
+      [#66](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/66),
+      [#77](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/77),
+      [#78](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/78),
+      [#80](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/80),
+      [#82](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/82),
+      [#83](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/83),
+      [#84](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/84),
+      [#85](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/85),
+      [#86](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/86),
+      [#87](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/87),
+      [#88](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/88),
+      [#89](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/89),
+      [#90](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/90),
+      [#91](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/91),
+      [#92](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/92),
+      [#93](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/93),
+      [#94](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/94),
+      [#95](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/95),
+      [#107](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/107),
+      [#108](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/108),
+      [#109](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/109),
+      [#120](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/120),
+      [#138](https://github.com/AY2425S1-CS2113-W14-4/tp/issues/138)
```

Comment:
While this is funny, might want to consider shortening this to "Maintained issues for the team", and just provide a few examples, because I think the PPP shouldn't be too long